packages/contracts/contracts/ChugSplashManager.sol

```diff
@@ -36,20 +36,22 @@ contract ChugSplashManager is
     Semver,
     IChugSplashManager
 {
-    bytes32 public constant EXECUTOR_ROLE = keccak256("EXECUTOR_ROLE");
+    bytes32 internal constant EXECUTOR_ROLE = keccak256("EXECUTOR_ROLE");
 
-    bytes32 public constant PROTOCOL_PAYMENT_RECIPIENT_ROLE =
+    bytes32 internal constant PROTOCOL_PAYMENT_RECIPIENT_ROLE =
         keccak256("PROTOCOL_PAYMENT_RECIPIENT_ROLE");
 
-    bytes32 public constant MANAGED_PROPOSER_ROLE = keccak256("MANAGED_PROPOSER_ROLE");
+    bytes32 internal constant MANAGED_PROPOSER_ROLE = keccak256("MANAGED_PROPOSER_ROLE");
 
-    bytes32 public constant NO_PROXY_CONTRACT_KIND_HASH = keccak256("no-proxy");
+    bytes32 internal constant NO_PROXY_CONTRACT_KIND_HASH = keccak256("no-proxy");
 
     /**
      * @notice Address of the ChugSplashRegistry.
      */
     ChugSplashRegistry public immutable registry;
 
+    ICreate2 public immutable create2;
+
```

Comment:
Nit: It would be nice to have some inline documentation here just noting why this logic is in a different contract.

Reply:
I'll add that when I go through the contracts and add docs to everything